src/app/index/index.component.css

```diff
@@ -133,47 +133,34 @@ footer {
   .dropdown-menu-box-homepage {
     margin-top: -15%;
   }
-  }
-

```

Comment:
@harshit98 Please thoroughly test. This is how your test link looks at 823px
![image](https://user-images.githubusercontent.com/20185076/29002769-95a3fd84-7ac7-11e7-9981-2e1888ff02ec.png)